File: app/src/main/java/com/jiujiu/mvvmTemplate/ui/main/MainActivityViewModel.java
package com.jiujiu.mvvmTemplate.ui.main;

import com.jiujiu.mvvmTemplate.data.DataManager;
import com.jiujiu.mvvmTemplate.ui.base.BaseViewModel;

import javax.inject.Inject;

public class MainActivityViewModel extends BaseViewModel {

    @Inject
    public MainActivityViewModel(DataManager dataManager) {
        super(dataManager);
    }

}

File: app/src/main/java/com/jiujiu/mvvmTemplate/ui/main/MainActivityViewModel.kt
package com.jiujiu.mvvmTemplate.ui.main

import com.jiujiu.mvvmTemplate.data.DataManager
import com.jiujiu.mvvmTemplate.ui.base.BaseViewModel

import javax.inject.Inject

class MainActivityViewModel @Inject constructor(
        dataManager: DataManager
) : BaseViewModel(dataManager)
